lib/travel_time/response.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'faraday'
+require 'travel_time/proto/utils'
```

Comment:
All library-related `require`s normally go [in the main library file](https://github.com/traveltime-dev/traveltime-sdk-ruby/blob/main/lib/travel_time.rb#L3-L7) (in your case, `lib/travel_time.rb`) rather than in individual files.